start-testnet.sh

```diff
@@ -0,0 +1,227 @@
+# First node has peers already set when collecting gentxs
+fsed 's#persistent_peers = ""#persistent_peers = "'$peer0','$peer2','$peer3'"#g' $n1cfg
+fsed 's#persistent_peers = ""#persistent_peers = "'$peer0','$peer1','$peer3'"#g' $n2cfg
+fsed 's#persistent_peers = ""#persistent_peers = "'$peer0','$peer1','$peer2'"#g' $n3cfg
+
+echo "Writing start commands"
+echo "$PEGGY --home home start --pruning=nothing > home.n0.log" >> $n0dir/startup.sh
+echo "$PEGGY --home home start --pruning=nothing > home.n1.log" >> $n1dir/startup.sh
+echo "$PEGGY --home home start --pruning=nothing > home.n2.log" >> $n2dir/startup.sh
+echo "$PEGGY --home home start --pruning=nothing > home.n3.log" >> $n3dir/startup.sh
+chmod +x $home_dir/*/startup.sh
+
+echo "Gathering keys for orchestrator"
+#--cosmos-key=<ckey>          The Cosmos private key of the validator
+#            --ethereum-key=<ekey>        The Ethereum private key of the validator
+#            --cosmos-legacy-rpc=<curl>   The Cosmos RPC url, usually the validator
+#            --cosmos-grpc=<gurl>         The Cosmos gRPC url, usually the validator
+#            --ethereum-rpc=<eurl>        The Ethereum RPC url, should be a self hosted node
+#            --fees=<denom>               The Cosmos Denom in which to pay Cosmos chain fees
+#            --contract-address=<addr>    The Ethereum contract address for Peggy, this is temporary
+
+echo "$n0name"_COSMOS_GRPC="http://$n0name:9090" >> $n0dir/orchestrator.env
+echo "$n0name"_COSMOS_RPC="http://$n0name:26657" >> $n0dir/orchestrator.env
+echo "$n0name"_COSMOS_KEY=$(jq .priv_key.value $n0cfgDir/priv_validator_key.json) >> $n0dir/orchestrator.env
+echo "$n0name"_DENOM=stake >> $n0dir/orchestrator.env
+echo "$n0name"_ETH_RPC=http://ethereum:8545 >> $n0dir/orchestrator.env
+
+echo "$n1name"_COSMOS_GRPC="http://$n1name:9090" >> $n1dir/orchestrator.env
+echo "$n1name"_COSMOS_RPC="http://$n1name:26657" >> $n1dir/orchestrator.env
+echo "$n1name"_COSMOS_KEY=$(jq .priv_key.value $n1cfgDir/priv_validator_key.json) >> $n1dir/orchestrator.env
+echo "$n1name"_DENOM=stake >> $n1dir/orchestrator.env
+echo "$n1name"_ETH_RPC=http://ethereum:8545 >> $n1dir/orchestrator.env
+
+echo "$n2name"_COSMOS_GRPC="http://$n2name:9090" >> $n2dir/orchestrator.env
+echo "$n2name"_COSMOS_RPC="http://$n2name:26657" >> $n2dir/orchestrator.env
+echo "$n2name"_COSMOS_KEY=$(jq .priv_key.value $n2cfgDir/priv_validator_key.json) >> $n2dir/orchestrator.env
+echo "$n2name"_DENOM=stake >> $n2dir/orchestrator.env
+echo "$n2name"_ETH_RPC=http://ethereum:8545 >> $n2dir/orchestrator.env
+
+echo "$n3name"_COSMOS_GRPC="http://$n3name:9090" >> $n3dir/orchestrator.env
+echo "$n3name"_COSMOS_RPC="http://$n3name:26657" >> $n3dir/orchestrator.env
+echo "$n3name"_COSMOS_KEY=$(jq .priv_key.value $n3cfgDir/priv_validator_key.json) >> $n3dir/orchestrator.env
+echo "$n3name"_DENOM=stake >> $n3dir/orchestrator.env
+echo "$n3name"_ETH_RPC=http://ethereum:8545 >> $n3dir/orchestrator.env
```

Comment:
Might be nice to break the above into a separate script where you could pass in a directory to populate as an arg.

Reply:
Yes, we should do that, and general prettyfication, after it works